test/integration/api/policies_controller_test.rb

```diff
@@ -7,7 +7,7 @@ def setup
     @provider = FactoryBot.create(:provider_account)
     @service = @provider.default_service
     login! @provider
-    Policies::PoliciesListService.expects(:call!)
+    Policies::PoliciesListService.expects(:call!).returns([])
```

Comment:
This smells. It mock an invalid behaviour, as `Policies::PoliciesListService` won't ever return an `Array`.

Reply:
@guicassolato the thing is that it either returns an array or raises an error, but it never returns `nil` as it was doing before this change... and adapting the real execution code to something that won't happen in real execution outside the tests just for this mock is even worse imo

Reply:
I think it never return an array.

Reply:
It should be an object, my bad!

Reply:
It should be a `String`.

P.S.: In Ruby, it's always an object, even when it's an `Array` ;)

Reply:
Right, it is a json/hash, not an Array, my bad :pray: So it should be its actual response :wink: but definitely not `nil` as it can never happen in real life.

Reply:
Sorry, a `Hash`, not a `String`

Reply:
> It should be a `String`.
> 
> P.S.: In Ruby, it's always an object, even when it's an `Array` ;)

Yes I meant a hash (javascript speaker here!)